Tools/autotest/arduplane.py

```diff
@@ -508,7 +508,8 @@ def fly_flaps(self):
             self.mavproxy.send('switch 1\n')  # auto mode
             self.wait_mode('AUTO')
             self.wait_ready_to_arm()
-            self.arm_vehicle()
```

Comment:
This shouldn't be changed - arming the vehicle here makes it self-contained and you can rearrange the tests.